packages/boot/test/bootstrapTests/test-net-ibc-upgrade.ts

```diff
@@ -111,66 +111,68 @@ test.serial('upgrade at many points in network API flow', async t => {
   const zoe: ZoeService = await EV.vat('bootstrap').consumeItem('zoe');
 
   const flow = entries({
-    bindPort: async () => EV(networkVat).bind('/ibc-port/'),
-    startServer: async boundPort => {
+    bindPorts: async label => {
+      const serverPort = await EV(networkVat).bind('/ibc-port/');
+      const clientPort = await EV(networkVat).bind('/ibc-port/');
+      return [label, { serverPort, clientPort }];
+    },
```

Comment:
I've taken the liberty of combining some steps that don't need to be interruptible for this test.